jina/resources/helloworld.flow.query.yml

```diff
@@ -6,11 +6,9 @@ with:
 pods:
   encode:
     uses: $RESOURCE_DIR/helloworld.encoder.yml
-    parallel: $PARALLEL
```

Comment:
Having `parallel` does not make sense, since the encoder saves the used matrix and having two in parallel means, they overwrite each others matrix. It works during querying, but half of the indexed documents will never retrieved, since they are encoded with a thrown away matrix.

Reply:
oh! good catch!